src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+    private String color;
+    private String name;
+
+    public Figure() {
+    }
+
+    public Figure(String color, String name) {
+        this.color = color;
+        this.name = name;
+    }
+
+    public String getInfo() {
+        return "Figure: "
+                + name
+                + " area: "
+                + getArea()
+                + " sq.units"
+                + " color: "
+                + color;
+    }
```

Comment:
You don't need this method. You just override toString in all shape classes for this